scenes/strategic-point-menu.gd

```diff
@@ -6,7 +6,8 @@ signal tower_bought(tower)
 
 func _ready() -> void:
 	for node in get_children():
-		node.connect("slot_activated", self, "_on_Slot_pressed")
+		if not node is Sprite :
+			node.connect("slot_activated", self, "_on_Slot_pressed")
```

Comment:
Here (and in similar occurrences), it would be better to use `if node is Slot`. There a 2 advantages:
1. more readable
2. Godot offers autocompletion for Slot's methods and properties because, inside the `if` statement, the `node` variable is a `Slot` type